src/java.management/share/classes/javax/management/modelmbean/DescriptorSupport.java

```diff
@@ -201,12 +201,16 @@ public DescriptorSupport(DescriptorSupport inDescr) {
      * @exception XMLParseException XML parsing problem while parsing
      * the input String
      * @exception MBeanException Wraps a distributed communication Exception.
+     * @deprecated This constructor exists for historical reasons.  It enables
+     * reading from XML, which is unnecessary.
```

Comment:
I wonder if we should say instead "Reading from XML if required should be implemented externally".